lib/index.js

```diff
@@ -204,13 +207,9 @@ module.exports.render = function(options) {
   var importer = options.importer;
 
   if (importer) {
-    options.importer = function(file, prev, key) {
+    options.importer = function(file, prev, bridge) {
       function done(data) {
-        console.log(data); // ugly hack
```

Comment:
Brilliant work! and thanks for explaining it!! :100: 

We were using `uv_mutex` and it was later we switched to std mutex. I had no idea that we can check already semaphored cond var in predicate.  :D

Everyday learning new things on GitHub.